stories/Autosuggest.js

```diff
@@ -51,86 +58,81 @@ storiesOf('Organisms|Autosuggest', module)
-                }
-            };
+            if (numOfItems === 1) {
+                return SUGGESTION_TO_STRING(store.get('selectedSuggestions')[0]);
+            }
 
-            const onBlur = () => {
-                console.log('onBlur was called');
-                setInputValue('');
-            };
+            return `${SUGGESTION_TO_STRING(store.get('selectedSuggestions')[0])} + ${numOfItems -
+                1} more`;
+        };
 
-            const onClearAllSelected = () => {
-                console.log('onClearAllSelected was called');
-                setSelectedSuggestions([]);
-            };
+        const onSelectionChange = item => {
+            console.log(`onSelectionChange was called with {name: ${item.name}}`);
+            if (store.get('selectedSuggestions').includes(item)) {
+                const newSelection = store
+                    .get('selectedSuggestions')
+                    .filter(el => el.name !== item.name);
+                store.set({ selectedSuggestions: newSelection });
+            } else {
+                store.set({ selectedSuggestions: [...store.get('selectedSuggestions'), item] });
+            }
+        };
 
-            return (
-                <Autosuggest
-                    selectedSuggestions={
-                        boolean('Add selectedSuggestions', true) ? selectedSuggestions : undefined
-                    }
-                    selectedPlaceholder={
-                        boolean('Add selectionPlaceholder', true)
-                            ? getSelectedPlaceholder()
-                            : undefined
-                    }
-                    isLoading={boolean('Loading', false)}
-                    showClearButton={boolean('Show clear button', true)}
-                    isMultiselect={boolean('Multiselect', true)}
-                    isProminent={boolean('Use prominent styling', true)}
-                    inputPlaceholder={text('Input placeholder', 'Select something...')}
-                    noSuggestionsPlaceholder={text('No suggestions', 'No suggestions found...')}
-                    clearTitle={text('Remove button label', 'Clear')}
-                    getSuggestions={getSuggestions}
-                    suggestionToString={SUGGESTION_TO_STRING}
-                    onBlur={onBlur}
-                    onSelectionChange={onSelectionChange}
-                    onClearAllSelected={onClearAllSelected}
-                    onInputValueChange={onInputValueChange}
-                    style={style}
-                />
-            );
+        const onBlur = () => {
+            console.log('onBlur was called');
+            store.set({ inputValue: '' });
         };
 
-        Implementation.displayName = 'Implementation';
+        const onClearAllSelected = () => {
+            console.log('onClearAllSelected was called');
+            store.set({ selectedSuggestions: [] });
+        };
 
-        return <Implementation style={{ width: '650px' }} />;
```

Comment:
It looks better in storybook if we keep the style. you can add it directly to Autosuggest

Reply:
Are you mean to add this style just in the last story  **Example implementation** or to all **Autosuggest**?